cmd/service.go

```diff
@@ -143,20 +146,55 @@ func runServiceStartCmd(cmd *cobra.Command, args []string) {
 
 	db.EnsureDBInstalled()
 
+	info, err := db.GetStatus()
+	if err != nil {
+		utils.ShowErrorWithMessage(err, "could not fetch service information")
+		return
+	}
+
+	if info != nil {
+		if info.Invoker == db.InvokerService {
+			fmt.Println("Service is already running")
+			printStatus(info)
+			return
+		}
+		// check that we have the same port and listen parameters
+		if port != info.Port {
+			utils.ShowError(fmt.Errorf("cannot convert service - port does not match with running instance"))
+			return
+		}
+		if listen != info.ListenType {
+			utils.ShowError(fmt.Errorf("cannot convert service - listen binding does not match with running instance"))
```

Comment:
can we display the current and requested binding

Reply:
updated the messagings in the failures to be a little but more descriptive